### PR TITLE
Complete milestone for project

### PR DESCRIPTION
**Pull request by obed300**
In the project we added below functions;

**Logic**
Each class from the [UML class diagram] were created in a separate .rb file.
All associations (1-to-many relationships) and aggregations (parent-child relationships) between classes are implemented.
All properties visible in the diagram are defined and set up in the constructor method.
All methods visible in the diagram are implemented:
add_item method (in all classes that have that method)
takes an instance of the Item class as an input.
adds the input item to the collection of items.
adds self as a property of the item object (by using the correct setter from the item object).
can_be_archived?() in the Item class
returns true if published_date is older than 10 years.
otherwise, it returns false.
move_to_archive() in the Item class
reuses can_be_archived?() method.
changes the archived property to true if the result of the can_be_archived?() method is true.
does nothing if the result of the can_be_archived?() method is false.
can_be_archived?() in the Book class
overrides the method from the parent class.
returns true if parent's method returns true OR if cover_state equals to "bad".
otherwise, it returns false.
can_be_archived?() in the MusicAlbum class
overrides the method from the parent class.
returns true if parent's method returns true AND if on_spotify equals true.
otherwise, it returns false.
can_be_archived?() in the Movie class
overrides the method from the parent class.
returns true if parent's method returns true OR if silent equals true.
otherwise, it returns false.
can_be_archived?() in the Game class
overrides the method from the parent class.
returns true if parent's method returns true AND if last_played_at is older than 2 years.
otherwise, it returns false.
Added unit tests for all implemented methods.
User interface
Created a main.rb file that serves as console app entry-point.
Our console app, at the start:
Presents the user with a list of options to perform.
Lets users choose an option.
If needed, asks for parameters for the option.
Has a way to quit the app.
The following options are also available:
List all books
List all music albums
List all movies
List of games
List all genres (e.g 'Comedy', 'Thriller')
List all labels (e.g. 'Gift', 'New')
List all authors (e.g. 'Stephen King')
List all sources (e.g. 'From a friend', 'Online shop')
Add a book
Add a music album
Add a movie
Add a game
All data are preserved by saving collections in .json files.

**Database**
**Added presentation video.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- LinkedIn: [@obed300](https://www.linkedin.com/in/obed-bamfo-4b152421b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- PRESENTATION VIDEO -->
+
+## Presentation Video <a name="presentation-video"></a>
+
+[Video](https://drive.google.com/file/d/1DTcLKxU8SM81VKbKYnSmqoNTlY-_RWGt/view)
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Great job recording a video, I was happy to see both of you appear and highlight the project pieces. However, there are a few requirements we need to meet for the video to reach an acceptable standard. Kindly record another video with a maximum of __5 minutes__. In order to meet this requirement, you should avoid one person sharing the screen. Each one of you should share your work (the first driver stops screen sharing and the second person starts sharing).

video

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class Main
+  def menu_options
+    "
+      1 - list all books
+      2 - list all music albums
+      3 - list all games
+      4 - list all genres
+      5 - list all labels
+      6 - list all authors
```
Comment (elisha2kyakpo1) [inline main.rb:11]: - [ ] I tried to list all authors by selecting optional 6, but the app started to request my first/last name, this is a bug that should be fixed. Kindly work on this and test if it works before submitting it for another review.

list-authors

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @obed300 and @josuerushanika, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 Almost the app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (josuerushanika) [reply README.md]: @elisha2kyakpo1 , thank you. We have completed another video presentation.

File: main.rb (hunk 2 above)
Comment (josuerushanika) [reply main.rb:11]: It's fixed

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi team 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
Thank you @codepantha

**Pull request merged**